examples/stats/web_client_monitoring/app.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * An Express application to receive the web metrics and send to Stackdriver
+ * with the opencensus API.
+ */
+
+"use strict";
+
+const express = require("express");
+const assert = require('assert');
+const process = require("process");
+const bodyParser = require('body-parser');
+const { Stats, MeasureUnit, AggregationType } = require('@opencensus/core');
+const { StackdriverStatsExporter } = require('@opencensus/exporter-stackdriver');
+
+const app = express();
+app.use(express.static("web_client"));
+app.use(bodyParser.json());
+
+// The project id must be provided. First, try the local environment
+const project = process.env.GOOGLE_CLOUD_PROJECT;
+assert(typeof project !== 'undefined' && project,
+  "Please set environment variable GOOGLE_CLOUD_PROJECT to the project id.");
+console.log(`Sending metrics data to project: ${project}`);
+
+// OpenCensus setup
+const stats = new Stats();
+const exporter = new StackdriverStatsExporter({projectId: project});
+stats.registerExporter(exporter);
+const mDNSTimeMs = stats.createMeasureDouble("web_client/dnsTime",
+                       MeasureUnit.MS,
+                       "DNS lookup time");
+const mConnectTimeMs = stats.createMeasureDouble("web_client/connctTime",
```

Comment:
It would be nice if these were instead folded into mLatencyMs as below but instead use tags e.g.
```js
const tagPhase = "phase";
const valueTLSNegotiation = "tls_negotiation";
const valueDNSLookup = "dns_lookup";
const valueLoad = "load";
```

and then for where we use `mDNSTimeMs`, just record mLatencyMs with tagKey `tagPhase` with value `valueDNSLookup`

Reply:
I have changed code according to this suggestion in the previous commit.

Reply:
Thanks, LGTM!